app/qml/account/MMWhichIndustryPage.qml

```diff
@@ -38,7 +38,8 @@ MMPage {
         { name: qsTr( "Natural resources" ), key: "natural", icon: __style.naturalResourcesIcon, colorx: __style.earthColor, color: "#4D2A24" },
         { name: qsTr( "Telecom" ), key: "telecom", icon: __style.telecommunicationIcon, colorx: __style.deepOceanColor, color: "#1C324A" },
         { name: qsTr( "Transportation" ), key: "transportation", icon: __style.transportationIcon, colorx: __style.skyColor, color: "#A6CBF4" },
-        { name: qsTr( "Water utilities" ), key: "water", icon: __style.waterResourcesIcon, colorx: __style.lightGreenColor, color: "#EFF5F3" }
+        { name: qsTr( "Water utilities" ), key: "water", icon: __style.waterResourcesIcon, colorx: __style.lightGreenColor, color: "#EFF5F3" },
```

Comment:
I checked the code and discovered two weird things:
 1. The `colorx` property is not used anywhere, it should be safe to remove it from all entries
 2. The `color` property uses colours with HEX strings instead of the common `__style` class. 

Can I ask you @Poulpator to fix both things (remove `colorx` and replace colour strings with data from `__style`)? 

```suggestion
        { name: qsTr( "Water utilities" ), key: "water", icon: __style.waterResourcesIcon, color: __style.xyzColor },
```